### ISSUE TITLE
Update kustomize in kubectl

### ISSUE THREAD

**Issue by dneilson** [kubernetes/kubectl#818]
The documentation out there is not right for the version baked in kubectl at the moment.

It is incredibly frustrating attempting to use it :|

Can we have this updated so we can use more options to generating manifests?

**Comment by seans3** [comment]
The current kustomize in kubectl is 2.0.3.
kustomize at head is above that.

/sig cli

**Comment by seans3** [comment]
The current challenge with updating kustomize in kubectl is the different versions of common dependencies. This challenge will be solved when kubectl completely removed from core kubernetes.

**Comment by mool** [comment]
It would be great to have the latest kustomize version in kubectl :-)

**Comment by brianpursley** [comment]
/remove-kind documentation

Sorry, I think I misclassified this. Sounds like this is not just a documentation thing.

**Comment by Gfeuillen** [comment]
+1 : the `kubectl` integration is one of the reasons we went for `kustomize`.

It would already be nice to be able to get the underlying `kustomize` version. @seans3 How did you arrive at this ?

**Comment by tylerauerbeck** [comment]
Are there any current plans to at least get kustomize on a 3.X release instead of remaining a full major release behind?

**Comment by Kampe** [comment]
The amount of issues present in 2.x.x is really annoying. Can we get this prioritized?

**Comment by Numblesix** [comment]
Is there any update its pretty hard to follow examples (as nee starter) just to find out that the kustomize version shipped in kubectl/oc is pretty outdated :(

**Comment by eddiezane** [comment]
/priority important-soon

Tied with #797

**Comment by monopole** [comment]
Tracking in the kustomize repo as https://github.com/kubernetes-sigs/kustomize/issues/1500
